ddpui/tests/integration_tests/test_airbyte_full_integration.py

```
@@ -0,0 +1,430 @@
+5. Running connection syncs and verifying their completion
+6. Fetching sync history and logs
+7. Tearing down workspace and associated entities
+"""
+
+import os
+import time
+import logging
+import pytest
+import requests
+from dotenv import load_dotenv
+from typing import List, Dict, Optional
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import

(F401)